src/@types/types.d.ts

```diff
@@ -175,6 +181,10 @@ export type ChromeModule = {
   analytics?: {
     APIKey?: string;
     APIKeyDev?: string;
+    amplitude?: {
```

Comment:
We don't currently support this in FEO/frontend-components validation, but I wanted to add it in for when we do